packages/platform-app/README.md

```diff
@@ -0,0 +1,3 @@
+# Platform App Types
+
+Provides typing for Platform Apps
```

Comment:
Can you add a bit more info like in https://github.com/Frontify/brand-sdk/blob/main/packages/guideline-blocks-settings/README.md ?